packages/utilities/src/utils/useId.ts

```diff
@@ -16,4 +16,4 @@ let idCounter = 0;
  *
  * @returns A generated ID that can be passed to accessibility attributes
  */
-export const useId = (id: any) => useReachId(id) || `id:${idCounter++}`;
+export const useId = (id?: any) => useReachId(id) || `id:${idCounter++}`;
```

Comment:
#578 inadvertently broke the API by requiring the user-provided ID parameter. Turning back to optional here.